package.json

```diff
@@ -18,7 +19,9 @@
     "deploy": "npm run build && node utils/deploy"
   },
   "devDependencies": {
-    "gh-pages": "^2.0.0"
+    "gh-pages": "^2.0.0",
+    "prettier": "^2.1.2",
+    "prettier-airbnb-config": "^1.0.0"
```

Comment:
I'd rather go with `eslint` + `prettier` as described in for example in https://gist.github.com/whoisryosuke/1bf81f43a79d2f93f5ba45cb4fe3a0a6 rather than installing such a package, that's not updated since more than year 😉

Reply:
Just added another config